src/config/configuration.ts

```diff
@@ -100,7 +100,7 @@ const configuration = () => {
       sample: sampleGroups.split(",").map((v) => v.trim()),
       samplePrivileged: samplePrivilegedGroups.split(",").map((v) => v.trim()),
       createJob: createJobGroups,
-      updateJob: updateJobGroups,
+      updateJob: statusUpdateJobGroups,
```

Comment:
Looks like this reverts Despina's changes

Reply:
Correction: https://github.com/SciCatProject/scicat-backend-next/pull/1460 isn't merged yet, but calling the local variable `updateJobGroups` will be more consistent once it merges